Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "FiniteDifferences"
 uuid = "26cc04aa-876d-5657-8c51-4c34ba976000"
-version = "0.9.2"
+version = "0.10.0"
```

Comment:
What makes this change breaking?

Reply:
The change of the `jacobian` API. If someone is currently using the `len` kwarg, that will now break.

Reply:
can we just deprecate it, rather than break it?

Reply:
We could, but all it'll do is completely ignore the `len` parameter. Is that something we want? (I'm not overly familiar with the conventions here)

Reply:
Yeah, that's not a breaking change.
Because there is only 1 valid value of `len` and passing anything else is an error anyway.
So  a deprecation that ignores `len` is nonbreaking

may have do do a manual deprecatation inside the `jacobian` function using `if` and `Base.depwarn`

Reply:
Done. If the emitted warning is appropriate, please resolve the conversation.